src/assets/styles/overrides.scss

```diff
@@ -40,6 +40,16 @@
   transform: translateY(-8px) scale(0.75);
 }
 
+// Disabled Text Fields
+.v-text-field.v-input--is-readonly {
+  pointer-events: none;
```

Comment:
Disabling pointer events means Vuetify doesn't make this look enabled when you click on it.